lib/entry/StateManager.js

```diff
@@ -20,7 +20,7 @@ module.exports = function(liveEntry) {
     let lastServerUpdate = null;
     let lastLiveTimestamp = new Date();
     let stateMachineMessagePattern = "Event '%s' raised for entry: Changing state from '%s' to '%s'";
-    let lastStreamInfoUpdateTime = 0;
+    let lastStreamInfoUpdateTime = new Date(1970,1,1);
```

Comment:
I think you meant new Date(0), this is more resistant to future changes to javascript implementation changes.